Dockerfile

```diff
@@ -1,33 +1,35 @@
 # syntax=docker/dockerfile:1
 ARG GO_VERSION
-FROM golang:${GO_VERSION}-alpine AS goimage
-
-FROM ghcr.io/ryboe/alpinecodespace:latest
-
-COPY --from=goimage /usr/local/go/ /usr/local/go/
-
-ENV PATH="${HOME}/go/bin:/usr/local/go/bin:${PATH}"
+FROM golang:${GO_VERSION}-alpine AS build
+ENV CGO_ENABLED="0"
```

Comment:
`CGO_ENABLED="0"` tells `go` to make statically linked binaries, so the Go utility binaries don't have any runtime dependencies.